src/main/java/com/rocs/infirmary/application/controller/inventory/InventoryController.java

```diff
@@ -53,6 +53,12 @@ public class InventoryController implements Initializable {
     private TableColumn<Medicine,String> itemTypeColumn;
     @FXML
     private TextField searchTextField;
+    @FXML
+    private CheckBox selectAllCheckbox;
+    @FXML
+    private Pagination pagination;
+    private static final int ROWS_PER_PAGE = 11;
+
```

Comment:
```suggestion
private FilteredList<Medicine> filteredList;
```

then on the first line of itemSearch replace

FilteredList<Medicine> filteredList = new FilteredList<>(medicine, b -> true);

with 

 filteredList = new FilteredList<>(medicine, b -> true);

so that you can use the filtered list properly in your pagination